README.md

```diff
@@ -26,7 +26,6 @@
 - plugin: <plugin-org/owner>/<plugin-repo-name>:compute.yml@development
   name: plugin-name
   id: plugin-name-command
-  type: github | worker
```

Comment:
Is this no longer a config requirement?

Reply:
Should be gone once https://github.com/ubiquity/ubiquibot-kernel/pull/57 is merged. The idea is that it can be deduced from the value within `plugin` (url pattern = worker, otherwise Action)

Reply:
Ahh that makes sense